packages/module/src/ChatbotConversationHistoryNav/ChatbotConversationHistoryNav.tsx

```diff
@@ -268,6 +264,28 @@ export const ChatbotConversationHistoryNav: FunctionComponent<ChatbotConversatio
             )}
           </DrawerActions>
         </DrawerHead>
+        <div className="pf-chatbot__input">
+          <Title headingLevel="h3">
+            <Icon size="lg">
+              <OutlinedClockIcon />
+            </Icon>
+            {'  '}
+            {title || 'Chat History'}
+          </Title>
+          {isLoading ? (
+            <LoadingState {...loadingState} />
```

Comment:
I'm seeing that the loading state looks a little funky:
<img width="400" alt="Screenshot 2025-07-03 at 11 33 55 AM" src="https://github.com/user-attachments/assets/78870c53-0824-4411-af13-b69e07330598" />

We want to do something like this rather than declaring the full loading state here:
```suggestion
            {!isLoading && handleTextInputChange && (
            <div className="pf-chatbot__input">
                <SearchInput
                  aria-label={searchInputAriaLabel}
                  onChange={(_event, value) => handleTextInputChange(value)}
                  placeholder={searchInputPlaceholder}
                />
               </div>
             )}
```